src/Bundler.js

```diff
@@ -66,7 +66,7 @@ class Bundler extends EventEmitter {
     const publicURL =
       options.publicUrl ||
       options.publicURL ||
-      '/' + Path.basename(options.outDir || 'dist');
+      '/' + Path.basename(options.outDir || '');
```

Comment:
Why set it to `options.outDir`? What does that provide? Why are these two related? I notice that this is the part that is not changed. I think we should change it, as I described in #932. To simply `/` by default.

Reply:
I didn’t want to change it too much, just fix a bug and prevent future issues regarding the default. Although i do agree with you, would be nice to have Devon’s opinion on these changes